Model/Customer.php

```diff
@@ -174,7 +173,15 @@ public function getLatestCard()
     protected function initializeObject()
     {
         if ($this->getId()) {
-            $this->customerAPI = $this->customerAPI->find($this->getId());
+            $customerAPI = $this->customerAPI->find($this->getId());
+
+            if ($customerAPI instanceof \Omise\Payment\Model\Api\Error) {
+                // instance of Error if merchants Omise account and key has been changed
```

Comment:
this comment is understandable?

Reply:
added a comment because I have no idea how I would name variables/function to make it (code) describing what it is and why it is here.